src/Features/CSharp/Portable/CodeFixes/Nullable/CSharpDeclareAsNullableCodeFixProvider.cs

```diff
@@ -202,6 +304,18 @@ static TypeSyntax TryGetSingleTypeArgument(TypeSyntax type)
                 }
                 return null;
             }
+
+            static TypeSyntax ParameterTypeSyntax(IParameterSymbol parameterSymbol)
```

Comment:
`TypeSyntax?` `TryGetParameterTypeSyntax`?

Reply:
Added nullable annotations to the file. Probably caught some nullable bugs :-)